### PR TITLE
Added support for Lumen

### PR DESCRIPTION
**Pull request by benjamincrozat**
Hello,

Here is a very simple PR to add support for Lumen to this lovely little package. Not much to say, I don't think I forgot anything. I'd suggest to update the repo's description to mention Lumen to improve its discoverability.

### CODE REVIEW

File: src/CorsServiceProvider.php [hunk 1]
```diff
@@ -28,6 +28,13 @@ public function boot()
 
     public function register()
     {
-        $this->mergeConfigFrom(__DIR__.'/../config/cors.php', 'cors');
+        if ($this->isNotLumen()) {
+            $this->mergeConfigFrom(__DIR__.'/../config/cors.php', 'cors');
+        }
+    }
+
+    protected function isNotLumen()
```
Comment (freekmurze) [inline src/CorsServiceProvider.php]: I like positive conditions better. Could you rename this to `isLaravel`?

File: src/CorsServiceProvider.php [hunk 2]
```diff
@@ -28,6 +28,13 @@ public function boot()
...
+    {
+        return ! preg_match('/lumen/i', app()->version());
```
Comment (freekmurze) [inline src/CorsServiceProvider.php]: Test for Laravel here.

**Review by freekmurze — CHANGES_REQUESTED** [review]

**Comment by freekmurze** [comment]
Thanks!

File: src/CorsServiceProvider.php [hunk 3]
```diff
@@ -28,6 +28,13 @@ public function boot()
...
     {
-        $this->mergeConfigFrom(__DIR__.'/../config/cors.php', 'cors');
+        if ($this->isLaravel()) {
+            $this->mergeConfigFrom(__DIR__ . '/../config/cors.php', 'cors');
+        }
+    }
+
+    protected function isLaravel()
+    {
+        return ! preg_match('/lumen/i', app()->version());
```
Comment (MichaelDeBoey) [inline src/CorsServiceProvider.php:38]: Why not do a positive check and check for `laravel`?

**Comment by benjamincrozat** [comment]
@MichaelDeBoey Because `app()->version()` on Laravel only returns the version number. Lumen does not. It returns `Lumen (5.6.0) (Laravel Components 5.6.*)` (something like that). It feels right to test it that way to me.

**Pull request merged**